src/FileCache.php

```diff
@@ -93,7 +90,7 @@ public function __construct(string $cachePath = '@runtime/cache', $serializer =
      */
     public function setCachePath(string $cachePath)
     {
-        $this->cachePath = Yii::getAlias($cachePath);
+        $this->cachePath = $cachePath;
```

Comment:
Dropping alias resolving here, as it should be done via configuration instead for Yii apps, correct?

Reply:
Yes.